examples/informers/IController.cs

```diff
@@ -0,0 +1,17 @@
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace informers
+{
+    /// <summary>
+    ///     Base interface for implementing controllers
+    /// </summary>
+    public interface IController
```

Comment:
Feels like this should be a bit more prescriptive. It doesn't do much right now.

Reply:
This is not final interface and is meant to explore what controllers interface should look like. I'll probably extend it to support shutdown hook as well. This is in a sample project so it's not part of public API.

Reply:
I see I didn't catch that. Makes sense.